src/DotNetWorker/OutputBindings/MethodReturnOutputBindingsInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.OutputBindings
+{
+    public class MethodReturnOutputBindingsInfo : OutputBindingsInfo
+    {
+        private const string ReturnBindingName = "$return";
+
+        private readonly IReadOnlyCollection<string> _bindingNames = new List<string>() { ReturnBindingName };
```

Comment:
Make this static as we're creating an instance of this per instance of this class, which gets created multiple times.